enterprise/internal/campaigns/reconciler_test.go

```diff
@@ -464,6 +466,94 @@ func TestReconcilerProcess(t *testing.T) {
 				externalState:  campaigns.ChangesetExternalStateClosed,
 			},
 		},
+		"reopening closed changeset without updates": {
+			currentSpec: &testSpecOpts{
+				headRef:   "refs/heads/head-ref-on-github",
+				published: true,
+
+				title: "title",
+				body:  "body",
+			},
+			previousSpec: &testSpecOpts{
+				headRef:   "refs/heads/head-ref-on-github",
+				published: true,
+
+				title: "title",
+				body:  "body",
+			},
+			changeset: testChangesetOpts{
+				publicationState:  campaigns.ChangesetPublicationStatePublished,
+				externalID:        githubPR.ID,
+				externalBranch:    githubPR.HeadRefName,
+				externalState:     campaigns.ChangesetExternalStateClosed,
+				createdByCampaign: true,
```

Comment:
```suggestion
				createdByCampaign: true,
				closing:           false,
```

It took me a minute to figure out why these would cause a reopen: I know the `closing` is implied here, but by comparison with the previous cases I think it'd be clearer to include it specifically.

Reply:
Agree!